idl/github.com/uber/cadence/cadence.thrift

```diff
@@ -174,6 +174,20 @@ service WorkflowService {
       3: shared.EntityNotExistsError entityNotExistError,
     )
 
+  /**
+  * RespondActivityTaskCompletedByID is called by application worker when it is done processing an ActivityTask.
+  * It will result in a new 'ActivityTaskCompleted' event being written to the workflow history and a new DecisionTask
+  * created for the workflow so new decisions could be made.  Use the 'taskToken' provided as response of
```

Comment:
This comment references taskToken. It looks like it was just copied from RespondActivityTaskCompleted

Reply:
Thanks for catching that! Updated.